client/js/screens/Search/SearchContainer.js

```diff
@@ -37,18 +43,22 @@ export default class SearchContainer extends Component {
         {({loading, error, data}) => {
           if (loading) return <Loader />;
           if (error) return <Text>Error :(</Text>;
-          const speakers = data.speakers.map(speaker => speaker.owner);
-          console.log(speakers);
-          if (data)
-            return (
-              <Search
-                events={data.events}
-                videos={data.videos}
-                speakers={data.speakers}
-                navigation={this.props.navigation}
-                route={this.props.route}
-              />
-            );
+          return (
+            <YoutubeDataContext.Consumer>
+              {value => {
+                const videos = value.videos.map(video => video.items);
+                return (
+                  <Search
+                    events={data.events}
+                    videos={videos}
```

Comment:
Is it necessary to create a new array? why not:

` 
<Search
videos={value.videos.items}
...
`